### PR TITLE
Moved migration script to eventuate-common. (Requires released version of eventuate-common)

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

**Comment by cer** [comment]
Please explain what you mean by "(Requires released version of eventuate-common)"

File: gradle.properties [hunk 1]
```diff
@@ -20,4 +20,6 @@ eventuatePlatformVersion=2020.4.BUILD-SNAPSHOT
 swaggerUiVersion=3.23.11
 eventuateExamplesBaseImageVersion=BUILD-5
 
+eventuateCommonVersion=0.13.0.RELEASE
```
Comment (cer) [inline gradle.properties:23]: This should be BUILD-SNAPSHOT

File: _build-and-test-all.sh [hunk 2]
```diff
@@ -29,20 +29,8 @@ ${dockerall}Up
...
-elif [ "${DATABASE}" == "postgres" ]; then
-  curl https://raw.githubusercontent.com/eventuate-foundation/eventuate-common/wip-db-id-gen/postgres/5.initialize-database-db-id.sql --output $migration_file --create-dirs
-  cat $migration_file | ./postgres-cli.sh -i
-else
-  echo "Unknown Database"
-  exit 99
-fi
-rm -f $migration_file
+export db_id_migration_repository=https://raw.githubusercontent.com/eventuate-foundation/eventuate-common
+curl -s https://raw.githubusercontent.com/eventuate-foundation/eventuate-common/master/migration/db-id/mssql/migration.sh &> /dev/stdout | bash
```
Comment (cer) [inline _build-and-test-all.sh:33]: Why is this in the 'mssql' directory?

Comment (dartartem) [reply _build-and-test-all.sh:33]: mistake, sorry

File: gradle.properties (hunk 1 above)
Comment (dartartem) [reply gradle.properties:23]: answered in status email

**Pull request merged**